pkg/experiment/ingester/memdb/head.go

```diff
@@ -21,10 +21,11 @@ import (
 )
 
 type FlushedHead struct {
-	Index    []byte
-	Profiles []byte
-	Symbols  []byte
-	Meta     struct {
+	Index                   []byte
+	Profiles                []byte
+	Symbols                 []byte
+	HasUnsymbolizedProfiles bool
```

Comment:
nit: Should we maybe rename this to `Unsymbolized` ? to make it look less like java and also to look more like the new label we add (I rememver it used to be `__has_unsymbolized_profiles__` but then we changed it)